src/py/aspen/config/config.py

```diff
@@ -29,7 +30,16 @@ def DEBUG(self):
 
     @property
     def SECRET_KEY(self):
-        return os.environ.get("SECRET_KEY")
+        # TODO: This probably needs to be sourced from secrets.  Somehow.  Who knows.
+        return "abcabcabc"
+
+    @property
+    def SESSION_TYPE(self):
+        # TODO: This needs to be updated to something more sane to persist sessions
+        # across restarts.
+        # https://flask-session.readthedocs.io/en/latest/#sqlalchemysessioninterface is
+        # probably a sane option.
+        return "filesystem"
```

Comment:
this is interesting. Do we need to persist sessions across restarts? I think if something like that happens sqlalchemy would automatically role back the session right?

Reply:
No, it wouldn't.  It'd force people to hit auth0 again, I believe.

Reply:
Ok so it looks like we don't need this.  Having a random value for `SECRET_KEY` is not great because it forces people to return to auth0, so we'll want some persistent value for that.